### PR TITLE
Update ControlPlaneKubeletLocalMode test for the feature gate being disabled

### PR DESCRIPTION
**Pull request by chrischdi**
Question: should we add/keep the a positive test? Or add the checks to another test to not duplicate? (maybe the regular one)?

Part of:
- https://github.com/kubernetes/enhancements/issues/4471

/hold

### CODE REVIEW

**Review by pacoxu — COMMENTED** [review]
/lgtm

File: kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-control-plane-local-kubelet-mode.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     testgrid-dashboards: sig-cluster-lifecycle-kubeadm
     testgrid-tab-name: kubeadm-kinder-control-plane-kubelet-local-mode-{{ dashVer .KubernetesVersion }}
     testgrid-alert-email: sig-cluster-lifecycle-kubeadm-alerts@kubernetes.io
-    description: "OWNER: sig-cluster-lifecycle (kinder); Uses kubeadm/kinder to create a cluster using the ControlPlaneKubeletLocalMode feature gate and run the kubeadm-e2e and the conformance suites"
+    description: "OWNER: sig-cluster-lifecycle (kinder); Uses kubeadm/kinder to create a cluster using the ControlPlaneKubeletLocalMode feature gate disabled and run the kubeadm-e2e and the conformance suites"
```
Comment (neolit123) [inline kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-control-plane-local-kubelet-mode.yaml:12]: we can skip the test infra update, i think. one less pr and it just matters for us the maintainers whether the fg is enabled or disabled.

**Review by neolit123 — COMMENTED** [review]
i think testing with the fg disabled until ga makes sense for this fg given the impact.
for the fg enabled case all regular jobs will exercise it.

**Comment by neolit123** [comment]
lgtm, please squash the commits to 1.
(the tide squash label doesn't work in this repo, afaik)

**Comment by chrischdi** [comment]
> lgtm, please squash the commits to 1. (the tide squash label doesn't work in this repo, afaik)

Done :-) PTAL

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve
